x-pack/plugins/apm/public/components/app/mobile/service_overview/index.tsx

```diff
@@ -134,8 +134,9 @@ export function MobileServiceOverview() {
                   values={{
                     feedbackLink: (
                       <EuiLink
+                        target={'_blank'}
                         data-test-subj="apmMobileServiceOverviewGiveFeedbackLink"
-                        href="https://ela.st/feedback-mobile-apm"
+                        href="https://docs.google.com/forms/d/e/1FAIpQLScQzsUFTp3ZULik65FZWrcAYYyNStOPgNkSZUQNyuyKfUybfA/viewform"
```

Comment:
Can't we have an ela.st link?

Reply:
oh! we certainly could have it. Let me update it